h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+import sys
+import traceback
+
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import os
+from h2o.exceptions import H2OResponseError
+
+
+def test_parser_svmlight_column_skip_not_supported():
+    print("Test that functions calling fail if skipped_columns is passed with svm file.")
+    # generate a big frame with all datatypes and save it to svmlight
+    nrow = 10
+    ncol = 10
+    seed = 12345
+
+    f1 = h2o.create_frame(rows=nrow, cols=ncol, real_fraction=0.5, integer_fraction=0.5, missing_fraction=0.2,
```

Comment:
I think this is also possible to simplify - we might not even need a response for this test but if we do create_frame has the option to generate a binary response.

I think for this type of test we just need one call to create_frame.

Reply:
I remove the second call and create only one frame. It works fine but I get warning that I am not able to track and handle.

The output of the test:

```
Test upload SVM file
Test OK, finished with H2OResponseError
Test import SVM file
Test OK, finished with H2OResponseError
Closing connection _sid_bf1c at exit
../..\h2o\h2o.py:780: UserWarning: [Lwater.parser.ParseWriter$ParseErr;@48c7a39e
  warnings.warn(w)
../..\h2o\h2o.py:780: UserWarning: [Lwater.parser.ParseWriter$ParseErr;@7ee8b863
  warnings.warn(w)
H2O session _sid_bf1c closed.
```

do you have any Idea?

Reply:
@michalkurka is this warning important?

Reply:
I think it deserves an investigation

Reply:
Fixed. The problem was setup of create_frame method missing_fraction=0.2 cause some missing value in the 0 column and svm parser report warning about it.